test/string.jl

```diff
@@ -169,18 +169,18 @@ end
     @test combine(gdf, :a) == combine(gdf, "a") ==
           combine(gdf, [:a]) == combine(gdf, ["a"])
 
-    @test combine("a" => identity, gdf, ungroup=false) ==
-          combine(:a => identity, gdf, ungroup=false)
-    @test combine(["a"] => identity, gdf, ungroup=false) ==
-          combine([:a] => identity, gdf, ungroup=false)
-    @test combine(nrow => :n, gdf, ungroup=false) ==
-          combine(nrow => "n", gdf, ungroup=false)
-
-    @test combine("a" => identity, gdf) == combine(:a => identity, gdf) ==
+    @test combine(gdf, "a" => identity, ungroup=false) ==
+          combine(gdf, :a => identity, ungroup=false)
+    @test combine(gdf, ["a"] => identity, ungroup=false) ==
+          combine(gdf, [:a] => identity, ungroup=false)
+    @test combine(gdf, nrow => :n, ungroup=false) ==
+          combine(gdf, nrow => "n", ungroup=false)
+
+    @test combine(gdf, "a" => identity) == combine(gdf, :a => identity) ==
           combine(gdf, "a" => identity) == combine(gdf, :a => identity)
-    @test combine(["a"] => identity, gdf) == combine([:a] => identity, gdf) ==
+    @test combine(gdf, ["a"] => identity) == combine(gdf, [:a] => identity) ==
           combine(gdf, ["a"] => identity) == combine(gdf, [:a] => identity)
-    @test combine(nrow => :n, gdf) == combine(nrow => "n", gdf) ==
+    @test combine(gdf, nrow => :n) == combine(gdf, nrow => "n") ==
           combine(gdf, nrow => :n) == combine(gdf, nrow => "n")
```

Comment:
These include duplicates calls.

BTW, we need deprecations for this, right?

Reply:
These deprecations will be extremely hard to do because of how much the rules have changed. I will add a method throwing an informative error.

Reply:
OK, but there are still duplicates. :-)

Reply:
I removed the duplicates